bcda/api/requests_test.go

```diff
@@ -506,50 +544,87 @@ func (s *RequestsTestSuite) TestDataTypeAuthorization() {
 	tests := []struct {
 		name string
 
-		cmsId        string
-		resources    []string
-		expectedCode int
-		acoConfig    *service.ACOConfig
+		cmsId           string
+		resources       []string
+		expectedCode    int
+		acoConfig       *service.ACOConfig
+		supplyAuthData  bool
+		breakBB         bool
```

Comment:
what's the difference here between `breakBB` and `breakBBInit`?

Reply:
breakBB will cause the call to get a patient to fail, since the basepath / connection isn't initialized when the BB client is returned.
breakBBInit breaks the actual client creation itself, since it induces an error condition where it's trying to pull a certificate/key that does not exist (control char).